robyn/processpool.py

```diff
@@ -67,26 +67,18 @@ def spawn_process(
         server.add_header(key, val)
 
     for route in routes:
-        route_type, endpoint, handler, is_async, number_of_params, const = route
-        server.add_route(
-            route_type, endpoint, handler, is_async, number_of_params, const
-        )
+        route_type, endpoint, function, const = route
```

Comment:
If we are refactoring this code, can we please rename the `const` variable to `is_const` or `is_const_function`?

Reply:
Makes sense, I changed it to `is_const` where it is relevant.